### ISSUE TITLE
Voice commands?

### ISSUE THREAD

**Issue by tremby** [IrosTheBeggar/mstream-android-app#55]
Does this client support voice commands?

For example

- play music by <artist>
- play the song <song> by <artist>
- play the album <album> by <artist>
- pause music
- skip to next track
- seek back ten seconds
- play <genre> music

**Comment by IrosTheBeggar** [comment]
No, this app doesn't support voice commands.  It sounds like a cool idea though.  Will put that on the todo list

**Comment by tremby** [comment]
It's the one thing stopping me drop Google Play Music or Spotify -- need to call up songs in the motorcycle helmet on dull stretches of road!

**Comment by IrosTheBeggar** [comment]
Glad to hear you like it!  I'm leaving the docs for voice commands here just incase any other devs decide to pick this up:

https://developer.android.com/training/wearables/apps/voice.html

Just curious, how did you hear about this project?

**Comment by tremby** [comment]
I've not actually used it yet. I've been looking for an open source music streaming server for which there's a decent Android app, and which handles transcoding and such. I came across mStream in [this list](https://github.com/Kickball/awesome-selfhosted#audio-streaming) and it looked promising.

**Comment by tremby** [comment]
The docs you found mentions wearables a lot.

I found also this: https://developers.google.com/voice-actions/system/
